src/sensor.h

```diff
@@ -90,6 +91,17 @@ namespace librealsense
             return {};
         }
 
+        // Sometimes it is more efficient to prepare for large or repeating operations. Depending on the actual sensor
+        // type we might want to change power state or encapsulate small transactions into a large one.
+        virtual void prepare_for_bulk_operation() {}
+        virtual void finished_bulk_operation() {}
+        // And ROII, too:
```

Comment:
`RAII`?

Reply:
Right :) WIll fix, or just remove the line...